subprojects/docs/src/docs/userguide/working_with_files.adoc

```diff
@@ -150,6 +150,8 @@ Notice how we specify the destination and name of the archive instead of an `int
 
 Each type of archive has its own task type, the most common ones being link:{groovyDslPath}/org.gradle.api.tasks.bundling.Zip.html[Zip], link:{groovyDslPath}/org.gradle.api.tasks.bundling.Tar.html[Tar] and link:{groovyDslPath}/org.gradle.api.tasks.bundling.Jar.html[Jar]. They all share most of the configuration options of `Copy`, including filtering and renaming.
 
+One notable difference between the `Copy` and `Sync` tasks and the archives is that duplicate files will not be allowed in archives anymore.
```

Comment:
```suggestion
One notable difference between the `Copy` and `Sync` tasks and the archive tasks is that duplicate files are not allowed in the latter by default.
```

I wonder if we should add a link to the DSL doc how the duplicate handling can be changed.

Reply:
That's already available in the upgrade notes, that should suffice for now, I think.

Reply:
The upgrade notes (as the release notes) are not something persistent. The user manual is. So if I am using Gradle 15.5, and I want to know how to include duplicate files in my JAR, where should I look? Linking to the DSL doc for `AbstractArchiveTask.duplicatesStrategy` should be enough.

Reply:
Done, please review and recommend alternative, if you feel it's needed.
Also https://docs.gradle.org/current/dsl/org.gradle.api.tasks.bundling.AbstractArchiveTask.html#org.gradle.api.tasks.bundling.AbstractArchiveTask:duplicatesStrategy

claims
> Default with java plugin:
> null

but upgrading_version_4 claims:
* For better compatibility with the Kotlin DSL, link:{javadocPath}/org/gradle/api/file/DuplicatesStrategy.html[CopySpec.duplicatesStrategy] is no longer nullable. The property setter no longer accepts null as a way
to reset the property back to its default value. Use DuplicatesStrategy.INHERIT instead.

If you think it's wrong, can you either push a fix or point me to the right place to fix it?

Reply:
I pushed a fix.